### DISCUSSION TITLE
How to remove the "Zellij" text in the status-bar?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [zellij-org/zellij#3552] [Q&A] [answered]
How can I remove the "Zellij" from the status bar?
![image](https://github.com/user-attachments/assets/af541098-709a-4c62-a6d0-29bbb065c796)
So far I've tried the below settings. I can't find any other related config option.
```
ui {
    pane_frames {
        hide_session_name true
    }
}
simplified_ui true
pane_frames false
default_layout "compact"
```

**Comment by giyany** [comment, marked as answer]
There is no native option to remove the Zellij branding.  The reason for this is that this helps other people discover the project. Zellij is an OSS project run by volunteers with no funding for advertizing or outreach, screenshots or the people looking over your shoulder are one of the main ways people find out about it.

Edit: English

**Reply by amirhhashemi** [reply]
I get why the branding is important for Zellij. I’d love to help spread the word!

**Reply by survirtual** [reply]
This is really unfortunate, and pushes me back toward tmux.

A good utility doesn't need to put its name everywhere.  Hopefully at some point in the future, configuration options will exist that care more about the user experience instead of the egos of the creators and devs.

**Reply by Lockszmith-GH** [reply]
Wonderful, as tmux is an excellent tool as well.

Kidding aside, it's FOSS, compile a version without it if it is THAT important.
